src/engine/game/gui/ingame/equip.cpp

```diff
@@ -80,25 +78,71 @@ Equipment::Equipment(Game *game) : GameGUI(game) {
     loadBackground(BackgroundType::Menu);
 }
 
+void Equipment::bindControls() {
+    binding.lblCantEquip = &getControl<Label>("LBL_CANTEQUIP");
```

Comment:
I'd say we either need to change `getControl` to return a pointer, or add another function to return a pointer, e.g. `T *getControlPtr<T>(const std::string &tag);`